source/fundamentals/transactions.txt

```diff
@@ -63,40 +63,45 @@ MongoDB in the following sections of this guide:
 Transaction APIs
 ----------------
 
-Use the **Core API** to implement a transaction with the driver. To use the Core
-API, you declare the start and commit points of the transaction.
+The driver provides multiple APIs for performing transactions, including
+the **Core API** and Convenient Transaction API. To begin using the Core
+API, declare the start and commit points of the transaction.
+
+To see an example that uses the Convenient Transaction API, see the
```

Comment:
Issue:

I think the information that explains what's different about the Convenient Transaction and Core API is missing. As a user, I would want to know when to use one over the other.

Suggestion:

I think this page. given it is the only Fundamentals page about Transactions in this docs set, should describe the differences between the Core API and Convenient Transaction in detail. The Usage Example should probably only contain the minimum information needed to show an example of the syntax and output.

Reply:
made significant changes to move information around